### ISSUE TITLE
The missing files

### ISSUE THREAD

**Issue by auh-12** [davidnvq/grit#41]
Dear author. The object detector missing some important files for training in train_config.yaml. Can you provide those files ! Thanks
${oc.env:DATA_ROOT}/vg/annotations/train_ann_lmdb ${oc.env:HOME}/datasets/vg/annotations/train_objects.json ${oc.env:HOME}/datasets/vg/annotations/attribute2ind.json ${oc.env:HOME}/datasets/vg/annotations/oid2attr.json
${oc.env:HOME}/datasets/vg/annotations/test_objects.json  ${oc.env:HOME}/datasets/vg/annotations/test_coco.pkl  ${oc.env:HOME}/datasets/vg/annotations/val_objects.json  ${oc.env:HOME}/datasets/vg/annotations/val_coco.pkl 
${oc.env:HOME}/datasets/coco/annotations/anno_1848_val2017.json
${oc.env:HOME}/datasets/coco/annotations/coco_vgoiv6_class2ind.json

**Comment by quangvnai** [comment]
Please find here for all the files. 
In fact, you don't need `train_ann_lmdb`. Just check the code of Dataset and turn off them. 

https://drive.google.com/drive/folders/1c7eWTjrlo_UJuKH3GwHIvTYMwzTkbIRd?usp=share_link

**Comment by auh-12** [comment]
Thank you. When I was training, I encountered some errors, such as screenshots. Do you know how to solve them?
![image](https://github.com/davidnvq/grit/assets/134300659/d856599b-b7e4-475a-a0be-ba95d9842094)
![5615d9f00b09c4eeccc329e0fc7cba8](https://github.com/davidnvq/grit/assets/134300659/9a6c6a0b-7b8d-4420-905c-db42ce0a156c)

**Comment by quangvnai** [comment]
I think the paths to images are wrong. Please check it carefully.

**Comment by auh-12** [comment]
Thank you. When I read the paper, I found that when pretrain the object detector with the backbone, you finetuned the models
on object detection plus attribute prediction using Visual Genome for additional five epochs. Can you share the code to implement it?

**Comment by quangvnai** [comment]
@auh-12 You are right. The pretrained object detectors shared in this repo were finetuned on OD and Attribute prediction using VG. I think you can find the code in the following files:
* https://github.com/davidnvq/grit/blob/main/engine/det_solver.py
* https://github.com/davidnvq/grit/blob/main/train_detector.py
* https://github.com/davidnvq/grit/blob/main/configs/detection/train_config.yaml
However, please be cautious that you may need to fix some incompatible bugs. I didn't clean these code files well as OD is not the main purpose of our paper. I'm sorry for this inconvenience.
